### PR TITLE
Translations - Add missing Spanish sentences

### PR DESCRIPTION
**Pull request by PrincipeAgustin**
- Added missing Spanish translation lines.

### CODE REVIEW

File: addons/sys_core/stringtable.xml [hunk 1]
```diff
@@ -380,6 +381,7 @@
             <Chinese>天線拉直</Chinese>
             <French>Antenne redressée</French>
             <Portuguese>Antena endireitada</Portuguese>
+            <Spanish>Antena derecha</Spanish>
```
Comment (TheMagnetar) [inline addons/sys_core/stringtable.xml]:
```suggestion
            <Spanish>Antena enderezada</Spanish>
```

File: addons/sys_core/stringtable.xml [hunk 2]
```diff
@@ -564,6 +569,7 @@
             <Chinese>顯示架上通訊</Chinese>
             <French>Afficher les transmissions du rack</French>
             <Portuguese>Mostras transmissões do raque</Portuguese>
+            <Spanish>Mostrar tráfico de radios integradas</Spanish>
```
Comment (TheMagnetar) [inline addons/sys_core/stringtable.xml]:
```suggestion
            <Spanish>Mostrar transmisiones de la radio vehicular</Spanish>
```

**Review by TheMagnetar — CHANGES_REQUESTED** [review]

File: addons/sys_core/stringtable.xml (hunk 1 above)
Comment (PrincipeAgustin) [reply addons/sys_core/stringtable.xml]: "Antena enderezada" is a literal translation and isn't correct, a more correct one is "Enderezar antena" i think, that mean´s put the antenna straight

**Comment by PrincipeAgustin** [comment]
i take one of the two suggestions and i put a more context sentence, only to the missing ones, in the future maybe change all sentences for the correct context one.

**Comment by PrincipeAgustin** [comment]
@TheMagnetar any news?

**Pull request closed without merging**